packages/web/src/scss/components/Accordion/_theme.scss

```
@@ -0,0 +1,8 @@
+@use '@tokens' as tokens;
+
+$accordion-spacer: tokens.$space-500;
+$accordion-item-hover-color: tokens.$background-interactive-hover;
```

Correct me if I am wrong, but I think the convention is `$block-element-property-state`
```suggestion
$accordion-item-color-hover: tokens.$background-interactive-hover;
```